_rules/video-audio-alternative-1ec09b.md

```diff
@@ -127,8 +125,8 @@ A video element with a track element that contains incorrect descriptions.
 
 ```html
 <video controls>
-	<source src="../test-assets/rabbit-video/video.mp4" type="video/mp4" />
-	<source src="../test-assets/rabbit-video/video.webm" type="video/webm" />
+	<source src="../test-assets/rabbit-video/video.mp4" type="video/mp4">
+	<source src="../test-assets/rabbit-video/video.webm" type="video/webm">
 	<track
```

Comment:
`<track>` is also a void element, so it shouldn't have the end slash, similar to `<source>`